crates/telio-firewall/src/firewall.rs

```diff
@@ -371,26 +384,32 @@ impl StatefullFirewall {
     ) -> bool {
         let ip = unwrap_option_or_return!(P::try_from(buffer), false);
         let peer: PublicKey = public_key.into();
-        {
-            // whitelist read-lock scope
-            let whitelist = unwrap_lock_or_return!(self.whitelist.read(), false);
-
-            // Fasttrack, if peer is whitelisted - skip any conntrack and allow immediately
-            if whitelist.peer_whitelist.contains(&peer) {
-                telio_log_trace!(
-                    "Outbound IP packet is for whitelisted peer, forwarding: {:?}",
-                    ip
-                );
-                return true;
+        let proto = ip.get_next_level_protocol();
+
+        // whitelist read-lock scope
+        let whitelist = unwrap_lock_or_return!(self.whitelist.read(), false);
+
+        // Fasttrack, if peer is whitelisted - skip any conntrack and allow immediately
```

Comment:
A small suggestion for updating the comment
```suggestion
        // If peer is whitelisted - allow immediately
```

Reply:
done